tests/helpers.py

```diff
@@ -1354,7 +1354,7 @@ def verify_volume_deleted_in_backend(interface, image_uuid, pool_name=None):
         pool_name (str): Name of the rbd-pool if interface is CephBlockPool
 
     Returns:
-        bool: True if volume is not present. False if volume is present
+        bool: True if volume is present. False if volume is not present.
```

Comment:
Please add an empty line below.
Please remove '.' from the end of the sentence

Reply:
Done.